src/Core/src/Sts/Input/AssumeRoleRequest.php

```diff
@@ -175,6 +178,9 @@ public function getSerialNumber(): ?string
         return $this->SerialNumber;
     }
 
+    /**
+     * @return Tag[]
```

Comment:
Not really related to that PR, but definining such dockbloc helps users and static tools (phpstan/psalm) following variable assignments.

Reply:
Yes. We should do more of these